go/common/genesis/genesis.go

```diff
@@ -8,7 +8,7 @@
 // On the date above, in accordance with the Business Source License, use of
 // this software will be governed by the GNU Lesser General Public License v3.
 
-package io
+package genesis
```

Comment:
Not a real fan of the `genesis` term due to its much wider (and to some extend different) meaning in the context of a block chain.

If this package is needed (see other comments) we should think about a better name.